GoogleForCreators/web-stories-wp
# Editor: Added icon to rotation handle

## Context

This makes the rotation handle larger and adds a 28x28px icon to it as per conversation with @aaskedall (and not a 19x19px icon as per Figma).

## Relevant Technical Choices

I needed the icon as either a path or as a raw SVG, so added a new "section" to the exported icons in the design-system, that also exports a raw SVG. This is probably not the best way to do it, so maybe a subfolder instead - so the resource would be `Icons.svg.arrowCircle` (lowercase letter, because it is not a component)?

I will add a comment inline in the PR for discussion.

(currently I think Storybook is broken, because it assumes all exports from icons are react components, and this one is not)

## Todo

- [ ] Fix the export/import of raw SVG's.

## User-facing changes

| Before | After |
|-|-|
| ![Screen Shot 2022-01-11 at 16 25 25](https://user-images.githubusercontent.com/637548/149024121-3192eb27-8330-47e3-8bca-06867fb1cefb.png) | ![Screen Shot 2022-01-11 at 16 25 03](https://user-images.githubusercontent.com/637548/149024132-47782ab9-26e4-40be-85d8-ce3a9f650bb0.png) |

## Testing Instructions

This PR can be tested by following these steps:

1. Select an element on a page
2. Observe the rotation handle

## Checklist




---

Fixes #10105
